packages/dds/shared-object-base/src/serializer.ts

```diff
@@ -63,8 +64,8 @@ export interface IFluidSerializer {
 export class FluidSerializer implements IFluidSerializer {
 	private readonly root: IFluidHandleContext;
 
-	public constructor(private readonly context: IFluidHandleContext) {
-		this.root = this.context;
+	public constructor(private readonly runtime: IFluidDataStoreRuntime) {
```

Comment:
I assume this is a temporary/prototyping change?  Otherwise seems like it would kind of defeat the point of IFluidHandleContext

Reply:
i don't know if there is a point to IFluidHandleContext. if the runtime has all the parts, i think it would be fine to get rid of, as i'm all for collapsing concepts. that said, we might still need it for other use cases, i'm not sure.